common/changes/@rushstack/rig-package/bartvandenende-wm-support-json-schema-draft-07_2024-05-15-14-55.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@rushstack/rig-package",
+      "comment": "Update build dependencies",
+      "type": "patch"
```

```suggestion
      "comment": "",
      "type": "none"
```

Changes to `devDependencies` aren't relevant to a consumer of the package.